### PR TITLE
Add possibility to make branch in branch page

### LINKED ISSUE #15911: Add possibility to make new branches in web interface
- Gitea version (or commit ref): commit c3aaf5eafd91b881ca67ab51e0cc2fd17a5b7921
- Database (use [x]):
    - [ ] PostgreSQL
    - [ ] MySQL
    - [ ] MSSQL
    - [X] SQLite

Description:
---
There is the possibility to make a commit with some changes but there are no the possibility to make a new branch. The equivalent to
`git checkout -b new_branch_name`
Adding this feature will case changes only it templates and adding new js handler because all we need at backend is in the code now.

This is new look of branch in branches page:
image

This is new form for setting name of new branch:
image

### PR DESCRIPTION
**Pull request by Viktor-Yakovchuk**
Add possibility to make branch in branch page
in the area next to Download and Delete buttons.
It is a more intuitive place in the interface
compared to what is already there.

Fix #15911 

Signed-off-by: Viktor Yakovchuk <viktor@yakovchuk.net>

### CODE REVIEW

**Comment by Viktor-Yakovchuk** [comment]
I have posted screenshots in the issue #15911

File: options/locale/locale_ru-RU.ini [hunk 1]
```diff
@@ -1949,6 +1949,10 @@ branch.restore=Восстановить ветку '%s'
 branch.download=Скачать ветку '%s'
 branch.included_desc=Эта ветка является частью ветки по умолчанию
 branch.included=Включено
+branch.create_new_branch=Создать ветку из ветки:
```
Comment (lafriks) [inline options/locale/locale_ru-RU.ini]: do not add translations in PR

**Review by lafriks — DISMISSED** [review]

File: options/locale/locale_ru-RU.ini (hunk 1 above)
Comment (Viktor-Yakovchuk) [reply options/locale/locale_ru-RU.ini]: Thanks, fixed

**Comment by Viktor-Yakovchuk** [comment]
continuous-integration seams failed with problem not related to this patch
amd64 failed but arm passed

**Review by noerw — APPROVED** [review]

File: templates/repo/branch/list.tmpl [hunk 2]
```diff
@@ -102,6 +107,11 @@
 										{{end}}
 									</td>
 									<td class="two wide right aligned overflow-visible">
+										{{if and $.IsWriter (not $.Repository.IsArchived) (not .IsDeleted)}}
+											<div class="ui basic jump button icon poping up show-create-branch-modal" data-branch-from="{{EscapePound .Name}}" data-content="{{$.i18n.Tr "repo.branch.new_branch_from" (.Name)}}" data-variation="tiny inverted" data-position="top right" data-modal="#create-branch-modal" data-name="{{.Name}}">
```
Comment (zeripath) [inline templates/repo/branch/list.tmpl]:
```suggestion
											<div class="ui basic jump button icon poping up show-create-branch-modal" data-branch-from="{{EscapePound .Name}}" data-content="{{$.i18n.Tr "repo.branch.new_branch_from" .Name}}" data-variation="tiny inverted" data-position="top right" data-modal="#create-branch-modal" data-name="{{.Name}}">
```

**Review by zeripath — APPROVED** [review]

**Comment by zeripath** [comment]
Make lgtm

**Pull request merged**